cli/loader_lib.c

```diff
@@ -186,6 +193,9 @@ int load_repl(const char * exe_dir, int argc, char * argv[])
 // Empty DLL main entrypoint to silence warning
 #ifdef _OS_WINDOWS_
 int __stdcall DllMainCRTStartup(void* instance, unsigned reason, void* reserved) {
+    // Because we override DllMainCRTStartup, we have to manuall call our constructor methods
```

Comment:
```suggestion
    // Because we override DllMainCRTStartup, we have to manually call our constructor methods
```

Reply:
Is there a particular reason we don't call CRT_INIT here as expected (such as not linking in that particular support library)?
https://docs.microsoft.com/en-us/troubleshoot/cpp/use-c-run-time

Reply:
We don't link to that library at all, so I don't think we need to do this.